src/redux/login/loginSlice.js

```diff
@@ -0,0 +1,54 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const URL = 'http://localhost:3000/api/v1/login';
+
+const initialState = {
+  name: '',
+  ifSucceed: false,
+  ifLoading: false,
+  errors: null,
+};
+
+export const submitLoginForm = createAsyncThunk(
+  'login/submitLoginForm',
+  async (formData, { rejectWithValue }) => {
+    try {
+      const response = await axios.post(URL, formData);
+      return response.data;
+    } catch (e) {
+      return rejectWithValue(e.message);
+    }
+  }
+);
+
+const loginSlice = createSlice({
+  name: 'login',
```

Comment:
We need the user id to manage the place and reservation creation. I kindly recommend adding it.